src/Entity/PropertyId.php

```diff
@@ -64,7 +70,7 @@ public function getEntityType() {
 	 * @return string
 	 */
 	public function serialize() {
-		return json_encode( array( 'property', $this->serialization ) );
+		return json_encode( array( 'property', $this->getSerialization() ) );
```

Comment:
I suggest to not touch these "magic" serialization methods to much, and not discuss this unrelated issue here.